.github/workflows/deploy-action.yaml

```
@@ -21,7 +21,12 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm i --legacy-peer-deps
+    - run: npm i
+    - name: Load Config JS
+      env:
+        CC_ENV_LOCAL: ${{ secrets.CC_ENV_LOCAL }}
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)